site/docs/numbers/porting/csrLookupTool.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 id: csrLookupTool
 title: CSR Lookup Tool
-slug: /numbers/guides/csrLookupTool
+slug: /numbers/guides/porting/csrLookupTool
 description: How to lookup CSR records using the Bandwidth API
 keywords:
   - bandwidth
```

Comment:
"This guide will walk you through how to programmatically lookup Customer Service Records (CSRs) for Phone Numbers before porting into your Bandwidth account."